CHT/HeatFlux.C

```diff
@@ -31,25 +34,38 @@ void preciceAdapter::CHT::HeatFlux::write(double * buffer)
     {
         int patchID = patchIDs_.at(j);
 
+        //TODO: Ask for the locationsType, currently Triangles assumed
+
+        //Setup Interpolation object
```

Comment:
Add a few comments / a summary on what is happening here. Keep in mind that this is the first adapter we implement this, so we need to have it as reference.